app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -40,6 +40,12 @@ child :content_facet => :content_facet_attributes do
     node(:view_products) { user.can?("view_products") }
     node(:create_bookmarks) { user.can?("create_bookmarks") }
   end
+
+  node :image_mode do |content_facet|
+    content_facet.image_mode_host?
+  end
+  attributes :bootc_booted_image, :bootc_booted_digest, :bootc_available_image, :bootc_available_digest,
+             :bootc_staged_image, :bootc_staged_digest, :bootc_rollback_image, :bootc_rollback_digest
```

Comment:
Do we need this `image_mode` in hosts index rabl as well? May not need all the digest fields on the index..

Reply:
That's true, we can infer based on the bootc information.

Reply:
The only reason you may want to leave those in is if it'd be useful for users to be able to add a column to All Hosts list page with that field.